src/CommunityCommons/javasource/communitycommons/ORM.java

```diff
@@ -255,6 +258,10 @@ public static String getValueOfPath(IContext context, IMendixObject substitute,
 		}
 	}
 
+	private static boolean isFileDocument(IMendixObject object) {
+	    return Core.isSubClassOf(Core.getMetaObject(FileDocument.entityName), object.getMetaObject());
```

Comment:
Indent is inconsistent with the rest of the file, which seems to be using tabs.

Reply:
Changed.

Reply:
I'd love that to be configurable in Studio Pro

Reply:
Fully agree!